contrib/python/conan_helpers.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import os
+
+def parse_chromium_readme(file_path):
+    result = dict()
+
+    with open(file_path, 'r', encoding='UTF-8') as fd:
+        for line in fd:
+            parts = line.split(": ", 2)
+            if len(parts) != 2:
+                break
+
+            result[parts[0].lower()] = parts[1].strip()
+
+    return result
+
+
+# There are some typos in the chromium supplied license info.
```

Comment:
See the comment on docstring above - a function comment should go underneath the `def` and contained in `"""`

Reply:
Done. Thanks!